packages/git/src/browser/git-scm-provider.ts

```diff
@@ -151,15 +151,15 @@ export class GitScmProvider implements ScmProvider {
                 }
             }
         }
-        state.groups.push(await this.createGroup('index', 'Staged changes', state.stagedChanges, true));
+        state.groups.push(await this.createGroup('merge', 'Merged Changes', state.mergeChanges, true));
```

Comment:
`Merged Changes` -> `Merge Changes` would be even better.

Reply:
Makes sense: https://github.com/microsoft/vscode/blob/30379ab74b74f76b94c28872cac3a9b1a38ff82e/extensions/git/src/repository.ts#L676

Reply:
thanks for checking!